src/login/components/GroupManagement/NameForm.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { connect } from "react-redux";
+import { Field, reduxForm } from "redux-form";
+import Form from "reactstrap/lib/Form";
+import CustomInput from "../Inputs/CustomInput";
+import EduIDButton from "../../../components/EduIDButton";
+import { emptyValueValidation } from "../../app_utils/validation/emptyValueValidation";
+import i18n from "../../translation/InjectIntl_HOC_factory";
+
+let NameForm = (props) => {
+  const { handleSubmit, invalid, form, label, placeholder, helpBlock } = props;
+
+  return (
+    <Form id={`${form}-form`} role="form" onSubmit={handleSubmit}>
+      <fieldset id={`${form}-fieldset`} className="tabpane">
+        <Field
+          label={label}
+          component={CustomInput}
+          componentClass="input"
+          type="text"
+          name={form}
+          placeholder={placeholder}
+          helpBlock={helpBlock}
+        />
+      </fieldset>
+      <EduIDButton type="submit" className="settings-button" disabled={invalid}>
+        CREATE GROUP
+      </EduIDButton>
+    </Form>
+  );
+};
+
+NameForm = reduxForm({
+  validate: emptyValueValidation,
+})(NameForm);
+
+NameForm = connect(() => ({
+  initialValues: {},
+  enableReinitialize: true,
```

Comment:
TIPS: Add touchOnChange: true, touchOnBlur: true in NameForm. so we get active validations.

Reply:
I have added `touchOnChange: true` to get immediate validation. I have also changed the border-bottom colour to on `.is-valid` to (#28a745) the green used for validation (I have also made this a variable called $valid-green).
 
It seems like `touchOnBlur` sets the form to "touched" and I am not sure I need that in my case. According to [the redux-form docs](https://redux-form.com/6.0.0-alpha.4/docs/api/reduxform.md/) it seems to be set to `true` by default, so you probably don't have to specify it if you want to use it.

Reply:
:+1: